src/sas/system/version.py

```diff
@@ -1,4 +1,3 @@
-__version__ = "6.0.0a1"
-__release_date__ = "2023"
+__version__ = "6.0.0b1"
+__release_date__ = "2024"
 __build__ = "GIT_COMMIT"
-
```

Comment:
Same comment as for `releasese_automation.py` - What is the point of `__build__` ?